src/analysis_file.py

```diff
@@ -24,8 +24,28 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
                            Metadata(user, Metadata.get_call_location(), time.time()))
 
         # Set the list of keys to be exported and how they are to be handled when folding
-        export_keys = ["uid", consent_withdrawn_key]
+        export_keys = ["uid"]
         bool_keys = [consent_withdrawn_key]
+        # Export listening group bool keys in analysis files headers only when running kakuma_pipeline because
+        # dadaab does not have listening groups.
+        if pipeline_configuration.pipeline_name == "kakuma_pipeline":
+            bool_keys.extend(
+                [
+                         "repeat_listening_group_participant",
+                         "kakuma_s01e01_listening_group_participant",
```

Comment:
Wonder if it would be better to set these dynamically from the RQAs using the same logic as was given listening_groups.py

Reply:
updated thanks